src/AddIns/Uno.UI.MSAL.Wasm/WasmScripts/uno-ui-msal.d.ts

```diff
@@ -0,0 +1,6 @@
+declare namespace MSAL {
+    class WebUI {
+        static authenticate(urlNavigate: string, urlRedirect: string, title: string, popUpWidth: number, popUpHeight: number): Promise<string>;
```

Comment:
Codacy found an issue: [The class method 'authenticate' must be marked either 'private', 'public', or 'protected'](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5872302)